### PR TITLE
Add Technical Specification template

### PR DESCRIPTION
**Pull request by manuelfuenmayor**
*As requested in https://github.com/metanorma/mn-templates-iso/issues/2#issuecomment-725830165*

*Relevant updates were applied to iso-rice template as well*

### CODE REVIEW

File: technical-specification/sections/zz-references.adoc [hunk 1]
```diff
@@ -0,0 +1,16 @@
+
+[bibliography]
+== Bibliography
+
+* [[[isotr17755,ISO/TR 17755:2014]]] _ISO/TR 17755, Fire safety -- Overview of national fire statistics practices_
+
+* [[[iso13943,ISO 13943:2017]]]	_ISO 13943:2017, Fire Safety -- Vocabulary_
```
Comment (ronaldtse) [inline technical-specification/sections/zz-references.adoc]: There's some spacing issue (tab?) here, can you help fix?

It should be:
```
* [[[iso13943,ISO 13943:2017]]], _ISO 13943:2017, Fire Safety -- Vocabulary_
```

**Review by ronaldtse — CHANGES_REQUESTED** [review]

File: technical-specification/sections/zz-references.adoc (hunk 1 above)
Comment (manuelfuenmayor) [reply technical-specification/sections/zz-references.adoc]: Huh, my text editor wasn't showing me that up. Fixed.

Comment (manuelfuenmayor) [reply technical-specification/sections/zz-references.adoc]: I added the missing comma to other (ISO) references as well.

**Comment by ronaldtse** [comment]
Merging. Thanks @manuel489 !

**Pull request merged**